test/e2e/retina_e2e_test.go

```diff
@@ -35,7 +36,18 @@ func TestE2ERetina(t *testing.T) {
 		*common.KubeConfig = infra.CreateAzureTempK8sInfra(ctx, t, rootDir)
 	}
 
+	// Install Ebpf and XDP
+	installEbpfAndXDP := types.NewRunner(t, jobs.InstallEbpfXdp(common.KubeConfigFilePath(rootDir)))
+	installEbpfAndXDP.Run(ctx)
+
+	time.Sleep(10 * time.Minute)
```

Comment:
What is this sleep for? Is there any more predictable checks or things we can validate to end the wait?